README.md

```diff
@@ -215,7 +215,7 @@ Then populate the database with the initial data using
 python ./scripts/dev_cli.py db-import
 ```
 
-You may also use `python ./scripts/dev_cli.py db-import -g` if you wish to import the generated mock data.
+You may also use `python ./scripts/dev_cli.py db-import` if you wish to import the generated mock data.
```

Comment:
```suggestion
```
🤦‍♂️ I should have read above

Reply:
In fact I think this section needs to be moved, its only a development thing, perhaps in the `Using mock data for testing [Optional]`.